### PR TITLE
Models created

### PR DESCRIPTION
**Pull request by katsuu**

### CODE REVIEW

File: app/models/hangout.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Hangout < ApplicationRecord
+  belongs_to :user
+  belongs_to :hangout_chat
```
Comment (domferris) [inline app/models/hangout.rb]: Should this be has_one maybe? I believe hangout is the parent in this case but I could be wrong

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,77 @@
...
+    t.bigint "user_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_friend_groups_on_user_id"
+  end
+
+  create_table "friendships", force: :cascade do |t|
+    t.bigint "sender_id"
+    t.bigint "receiver_id"
+    t.string "approved", default: "Pending"
```
[the remark this answers is not in the document]
Comment (katsuu) [reply db/schema.rb]: Depends on the behaviour we want, if it's false then do we always keep the friend request open in friends requests? Would this list build up if user kept as false? I would think false in these terms would translate to deleting the friend request. Whatcha think?

File: db/schema.rb [hunk 3]
```diff
@@ -10,11 +10,77 @@
...
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["hangout_chat_id"], name: "index_hangout_messages_on_hangout_chat_id"
+    t.index ["user_id"], name: "index_hangout_messages_on_user_id"
+  end
+
+  create_table "hangouts", force: :cascade do |t|
+    t.time "start_time"
+    t.time "end_time"
+    t.date "date"
```
[the remark this answers is not in the document]
Comment (katsuu) [reply db/schema.rb]: For date? What would you like to do with the time?

File: db/schema.rb [hunk 4]
```diff
@@ -10,11 +10,77 @@
...
+    t.string "name"
+    t.bigint "user_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_friend_groups_on_user_id"
+  end
+
+  create_table "friendships", force: :cascade do |t|
+    t.bigint "sender_id"
+    t.bigint "receiver_id"
```
[the remark this answers is not in the document]
Comment (katsuu) [reply db/schema.rb]: Good shout, we can add that in!

File: app/models/hangout.rb (hunk 1 above)
Comment (katsuu) [reply app/models/hangout.rb]: Yep you're right! This was the one I was super iffy about. Both ways should be a has_one right? As in Hangout has one hangout chat and hangout chat has one hangout?

File: db/schema.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (katsuu) [reply db/schema.rb]: *update to datetime for start and end

**Pull request merged**